packages/ui/src/components/SubscriptionCta/SubscriptionCta.tsx

```diff
@@ -0,0 +1,48 @@
+import type { FormEvent } from 'react';
+import { twMerge } from 'tailwind-merge';
+
+import { Button } from '../Button';
+import { Input } from '../Input';
+
+export interface Props {
+    className?: string;
+    value: string;
+    onChange: (value: string) => void;
+    onSubmit: () => void;
+}
+
+export function SubscriptionCta({ className, value, onChange, onSubmit }: Props) {
+    function handleSubmit(event?: FormEvent<HTMLFormElement>) {
+        event?.preventDefault();
+
+        onSubmit();
+    }
+
+    return (
+        <div
+            className={twMerge(
+                'flex flex-col md:flex-row items-center py-12 md:py-24 px-6 md:px-20 gap-8 bg-gray-700',
+                className,
+            )}
+        >
+            {/* TODO: add translations */}
+            <h3 className="title-large w-full md:w-1/2 text-white">Get updates in your inbox</h3>
+            <form
+                className="flex flex-col sm:flex-row items-center w-full md:w-1/2 gap-4"
+                onSubmit={handleSubmit}
+            >
+                <Input
+                    className="w-full"
+                    inputClassName="w-full"
```

Comment:
Shouldn't input always be the full width of its container? IMO this should be encapsulated in the `Input` component, then you'll only control the overall component size.